styx-common/src/test/java/com/spotify/styx/storage/DatastoreStorageTest.java

```diff
@@ -734,6 +735,19 @@ public void runInTransactionShouldThrowIfDatastoreNewTransactionFails() throws E
     verify(transactionFunction, never()).apply(any());
   }
 
+  @Test
+  public void shouldReturnShardsForCounter() throws Exception {
+    storage.runInTransaction(tx -> {
+      tx.store(Shard.create(COUNTER_ID1, 0, 0));
+      tx.store(Shard.create(COUNTER_ID1, 1, 3));
+      return null;
+    });
+    final Map<Integer, Long> map = storage.shardsForCounter(COUNTER_ID1);
+    assertEquals(2, map.size());
```

Comment:
Minor nitpick: all the other tests use `assertThat` and `is`, for consistent and good readability of messages.

Reply:
That has been a question mark in my head for long time. What is the benefit of using assertThat?